Detections/AuditLogs/AdminPromoAfterRoleMgmtAppPermissionGrant.yaml

```diff
@@ -0,0 +1,74 @@
+id: f80d951a-eddc-4171-b9d0-d616bb83efdc
+name: Admin promotion after Role Management Application Permission Grant
+description: |
+  'This rule looks for a service principal being granted the Microsoft Graph RoleManagement.ReadWrite.Directory (application) permission before being used to add an Azure AD object or user account to an Admin directory role (i.e. Global Administrators). 
+  Ref : https://docs.microsoft.com/en-us/graph/permissions-reference#role-management-permissions, https://docs.microsoft.com/en-us/graph/api/directoryrole-post-members?view=graph-rest-1.0&tabs=http'
```

Comment:
Remove the en-us from the reference links, this is causing the validation failure.

Reply:
Thank you Shain! I always wondered what those error messages were :) Fixed.